src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -722,6 +999,87 @@ DevicesDiscoveryStarter& ConnectionHandlerImpl::get_device_discovery_starter() {
   return *this;
 }
 
+NonConstDataAccessor<SessionConnectionMap>
+ConnectionHandlerImpl::session_connection_map() {
+  return NonConstDataAccessor<SessionConnectionMap>(
+      session_connection_map_, session_connection_map_lock_);
+}
+
+SessionTransports ConnectionHandlerImpl::SetSecondaryTransportID(
+    uint8_t session_id,
+    transport_manager::ConnectionUID secondary_transport_id) {
+  SessionTransports st;
+
+  NonConstDataAccessor<SessionConnectionMap> session_connection_map_accessor =
+      session_connection_map();
+  SessionConnectionMap& session_connection_map =
+      session_connection_map_accessor.GetData();
+  SessionConnectionMap::iterator it = session_connection_map.find(session_id);
+  if (session_connection_map.end() == it) {
+    LOG4CXX_WARN(logger_,
+                 "SetSecondaryTransportID: session ID "
+                     << static_cast<int>(session_id)
+                     << " not found in Session/Connection map");
+    st.primary_transport = 0;
+    st.secondary_transport = 0;
+  } else {
+    st = it->second;
+
+    // The only time we overwrite an existing entry in the map is if the new
+    // secondary transport ID is 0xFFFFFFFF, which effectively DISABLES the
+    // secondary transport feature for the session, or if the new secondary
+    // transport ID is 0, which means a secondary transport has shut down
+    if (st.secondary_transport != 0 && secondary_transport_id != 0xFFFFFFFF &&
```

Comment:
Should include a constant for `0xFFFFFFFF`

Reply:
Fixed in commits b4c6b5e040783ccd8cb74d411024d3b8a19c9c84 and 8fda3ea7b563cc5d343dc6e9a4dbd094c1085659

Reply:
Fixed in commits b4c6b5e040783ccd8cb74d411024d3b8a19c9c84 and 8fda3ea7b563cc5d343dc6e9a4dbd094c1085659